### PR TITLE
Fixed local build.

### PR DESCRIPTION
**Pull request by mcrusch**
This included upgrading the gradle build to gradle 7, and adding the gradle wrappers.

Note: java.sh and picard had to be updated to accommodate the fact that we have a monolithic picard jar in the Docker image, but a lite picard jar with separate htsjdk for local use.

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```
Comment (adthrasher) [inline gradlew:1]: Does this need to be in the repo?

File: gradle/wrapper/gradle-wrapper.jar
Comment (adthrasher) [inline gradle/wrapper/gradle-wrapper.jar:1]: Does this need to be in the repo?

File: gradlew (hunk 1 above)
Comment (mcrusch) [reply gradlew:1]: My understanding of the best practice is to include gradle with the wrapper in the repo.  Besides eliminating the need for the user to install gradle, it also means your gradle version and build script are in sync.  So, if a user has multiple projects with gradle build scripts from different versions, they don't need to install multiple versions of gradle and choose between them when they build the project.

File: gradle/wrapper/gradle-wrapper.jar
Comment (mcrusch) [reply gradle/wrapper/gradle-wrapper.jar:1]: See comment on gradlew

**Review by adthrasher — APPROVED** [review]

**Pull request merged**